include/api.php

```diff
@@ -635,7 +635,7 @@ function api_get_user(App $a, $contact_id = Null, $type = "json"){
 
 		$r = q("SELECT count(*) as `count` FROM `contact`
 				WHERE  `uid` = %d AND `rel` IN ( %d, %d )
-				AND `self`=0 AND NOT `blocked` AND `hidden`=0",
+				AND `self`=0 AND NOT `blocked` AND NOT `pending` AND `hidden`=0",
```

Comment:
`AND `self`=0 AND NOT `blocked` AND NOT `pending` AND `hidden`=0",` I see 2 different ways of handling "boolean" values (or flags).